entrypoints/content/pages/settings/AppearanceForm.tsx

```diff
@@ -0,0 +1,154 @@
+// Packages:
+import { useEffect } from 'react'
+import useUserPreferences from '../../hooks/useUserPreferences'
+import { cn } from '../../lib/utils'
+
+// Typescript:
+import { Theme, Visibility } from 'types/user-preferences'
+
+// Components:
+import {
+  Select,
+  SelectContent,
+  SelectGroup,
+  SelectItem,
+  SelectTrigger,
+  SelectValue,
+} from '../../components/ui/select'
+
+// Functions:
+const AppearanceForm = () => {
+  // Constants:
+  const {
+    // TODO: Disable all inputs across /settings when isUserPreferencesLoading is true.
+    isLoading: isUserPreferencesLoading,
+    loadUserPreferences,
+    appearance,
+    setTheme,
+    setVisibility,
+  } = useUserPreferences()
+
+  // Effects:
+  useEffect(() => {
+    loadUserPreferences()
+  }, [])
```

Comment:
i saw this across the 3 forms and it doesn't make sense
the parent `settings/index.tsx` already loads the state from BE, why are we loading it again and again on every form?
logically, it'll lead to more BE calls -> higher cost incurred
to optimize this, remove this effect and the effect import from all 3 files and only have it in the `settings/index.tsx`